### DISCUSSION TITLE
Should we use adjusted p-value (q value) if we do not conduct a a multi-group comparisons?

### DISCUSSION THREAD

**Discussion by xnysound** [FrederickHuangLin/ANCOMBC#203] [Q&A]
Hi Lin, 

Should we use adjusted p-value (q value) if we DO NOT perform multi-group comparisons (e.g., global test, pairwise directional test, Dunnett's type of test, or trend test)? I am trying to use ANCOM-BC to estimate the log-fold change in species per 1-SD increment in variable X (a continuous varaible): 
out = ANCOMBC::ancombc(phyloseq = Filtered_newphylo, 
                       formula = "scale(X) + age + sex + bmi + physical_activity",       
                       p_adj_method = "fdr", prv_cut = 0, lib_cut = 0,
                       group = NULL, struc_zero = FALSE, neg_lb = FALSE, tol = 1e-5,
                       max_iter = 100, conserve = FALSE, alpha = 0.05, global = FALSE)

Another question is: If I am trying to use ANCOM-BC to estimate the log-fold change in species per 1-SD increment in variable A, B, C (all continuous variables), but do not want to compare between these three variables,  can I loop over a vector M (= A, B, C)? I think in this case, I should not use "group". 
out = ANCOMBC::ancombc(phyloseq = Filtered_newphylo, 
                       formula = "scale(M) + age + sex + bmi + physical_activity",       
                       p_adj_method = "fdr", prv_cut = 0, lib_cut = 0,
                       group = NULL, struc_zero = FALSE, neg_lb = FALSE, tol = 1e-5,
                       max_iter = 100, conserve = FALSE, alpha = 0.05, global = FALSE)
Then, I can use the result to plot a heat map. 

Thank you very much!

**Comment by FrederickHuangLin** [comment]
Hello @xnysound,

Certainly, utilizing adjusted p-values is a sound practice as it serves as a safeguard against false positives resulting from multiple testing. This applies not only to multiple comparisons but also to the broader spectrum of multiple testing across taxa. The exception would be in cases with notably small sample sizes where prioritizing statistical power becomes paramount.

I gather that your objective was to determine the effect size associated with a 1 SD increase in a continuous variable. In this context, standardizing the variable is a straightforward approach, ensuring that each unit corresponds to a 1 SD change. Your observation that there is no need to specify the `group` argument is indeed accurate.

Best regards,
Huang

**Reply by xnysound** [reply]
Thank you Huang for clarifying this!
